.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = E402,W504
+exclude = .git,__pycache__,doc/source/conf.py,build,dist,versioneer.py,caproto/_headers.py,caproto/tests/test_pyepics_compat.py,caproto/_version.py
+max-line-length = 115
```

Comment:
My concern here is that the line length here is 115, but in [.travis.yml](https://github.com/NSLS-II/nslsii/blob/d3f942cda8eac713ac625dbcf4285e108c04f154/.travis.yml#L24) it's set to 79 via command-line arg (which most likely has a priority). I think we should have a single source of truth here, so that only one configuration is controlling the settings in both development environments and in the CI systems.

Reply:
I wonder why it was set to `79` in `.travis.yml` since that is the default anyway. @ke-zhang-rd, I think the right path here is to remove `--max-line-length=79` from [this line](https://github.com/NSLS-II/nslsii/blob/master/.travis.yml#L24) as part of this PR.

Reply:
I agree with the plan.